x-pack/legacy/plugins/apm/server/lib/transactions/get_transaction_by_trace/index.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  PROCESSOR_EVENT,
+  TRACE_ID,
+  PARENT_ID
+} from '../../../../common/elasticsearch_fieldnames';
+import { Transaction } from '../../../../typings/es_schemas/ui/Transaction';
+import { Setup } from '../../helpers/setup_request';
+import { ProcessorEvent } from '../../../../common/processor_event';
+
+export async function getTransactionByTraceId(traceId: string, setup: Setup) {
```

Comment:
This should be called `getRootTransactionByTraceId` to signal it's getting the very first transaction in the trace.